### PR TITLE
Update to Jenkins 1.619; Java 7; new maintainers.

### PR DESCRIPTION
**Pull request by astroilov**

### CODE REVIEW

**Comment by astroilov** [comment]
The check failures appear to be due to findBugs warnings in code not affected by this pull request. Specifically, an existing call to Hudson.getInstance(), part of the boilerplate of all() and getDescriptor(), which is not null-checked.

While this is extremely unlikely to be a "real" bug, I'll add null-check code to remove extraneous warnings.

File: src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java [hunk 1]
```diff
@@ -475,7 +484,10 @@ private void performUploadWithRetry(Executor executor, Storage service,
         return;
       } catch (IOException e) {
```
Comment (tcnghia) [inline src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java]: I believe Java 7 has multiple catch to trim a few lines here

File: src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java [hunk 2]
```diff
@@ -350,6 +351,13 @@ public Void call() throws UploadException {
                   remoteCredentials, uploads, listener);
               return (Void) null;
             }
+
+            @Override
+            public void checkRoles(RoleChecker checker)
```
Comment (tcnghia) [inline src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java]: unbreak line?

**Comment by tcnghia** [comment]
LGTM, only a few minor cosmetic comments.

File: src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java (hunk 1 above)
Comment (astroilov) [inline src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java]: Unfortunately, the precise semantics of multiple catch don't work for what we want to do with this exception.

File: src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java (hunk 2 above)
Comment (astroilov) [inline src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java]: Just a bit too long.

**Pull request merged**